packages/core/contracts/oracle/implementation/DesignatedVotingV2Factory.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../../common/implementation/Withdrawable.sol";
+import "./DesignatedVotingV2.sol";
+
+/**
+ * @title Factory to deploy new instances of DesignatedVotingV2 and look up previously deployed instances.
+ * @dev Allows off-chain infrastructure to look up a hot wallet's deployed DesignatedVoting contract.
+ */
+contract DesignatedVotingV2Factory is Withdrawable {
```

Comment:
Why is `DesignatedVotingV2Factory` `Withdrawable`?

Reply:
it was also in v1 so I left it the same here. I dont think it really needs this inheritance path but I wanted to keep things as similar to what we had before and can remove/add new logic in subsequent PRs.